api/pkg/apis/v1alpha1/providers/states/k8s/k8s.go

```diff
@@ -511,6 +676,82 @@ func (s *K8sStateProvider) Get(ctx context.Context, request states.GetRequest) (
 	return ret, nil
 }
 
+func (s *K8sStateProvider) GetLatest(ctx context.Context, request states.GetRequest) (states.StateEntry, error) {
```

Comment:
We should not extend StateProvider to have this GetLatest() function, as the state provider abstracts a key-value store without versioning concept.